src/Card/README.md

```diff
@@ -129,22 +129,25 @@ link styling to make its content appear as a regular text.
   const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
 
   return (
-    <Hyperlink destination="https://www.edx.org">
```

Comment:
We should also change description of this section because right now it says that we wrap `Card` with a link but we use `as` prop instead.